### ISSUE TITLE
Property 'select' does not exist on type 'Multiselect'

### ISSUE THREAD

**Issue by cnakh** [vueform/multiselect#287]
I use with Vue 3 with TypeScript as code below

```
<template>
    <Multiselect
          valueProp="id"
          label="name"
          :options="getCategories"
          :searchable="true"
          ref="selectCategory"
    />
</template>

<script setup lang="ts">
import Multiselect from "@vueform/multiselect";
...
const selectCategory = ref<InstanceType<typeof Multiselect>>();
...
</script>
```

When I call `select` method it work fine in development, but when I build it get error

```
selectCategory.value.select(data);
```

```
Property 'select' does not exist on type 'Multiselect'.
```

**Comment by n14s** [comment]
I'm also interested to know how to use the API with Vue3 composition API and `script setup`

**Comment by antpngl92** [comment]
I had a similar error trying to access `selectCategory.value.isOpen` boolean. 

To fix that I created a mock ref type: 

```ts
type MultiselectRef = InstanceType<typeof Multiselect> & {
  isOpen: boolean;
}
```

in your case will be 

```ts
type MultiselectRef = InstanceType<typeof Multiselect> & {
  isOpen: (data: object | string | whatever) => void;
}
```

@adamberecz   Perhaps a type definition with all attributes of the ref object would fix further similar issues?

**Comment by antpngl92** [comment]
> Could anyone reliably help me with TS issues to put everything in place once and for all in both Vue 2 & 3? If so, please let me know!

I can do it, will open a PR at some point today.

**Comment by tholu** [comment]
@antpngl92 Thanks for your PR!
@n14s Did you manage to get that working somehow? Even with the mock ref type from @antpngl92 I'm getting  `multiselect.value?.select is not a function`, already in development when trying to call the select method.

```ts
type MultiselectRef = InstanceType<typeof Multiselect> & {
    select: (data: object | string) => void
}

const multiselect = ref<MultiselectRef>()

[...]

multiselect.value?.select(data)
```

The reason for trying to call the select method is that the value shown in the multiselect is not updated when using `setFieldValue()` from vee validate (still shows the placeholder).

**Comment by crobinson-expl** [comment]
[This PR](https://github.com/vueform/multiselect/pull/321) fixes an issue I'm having with TypeScript and the `open` API method.  @adamberecz, what is preventing the PR from being approved and merged?

**Comment by adamberecz** [comment]
This should now be fixed in `2.6.3`. Let me know if you still encounter any issues. Thanks for the patience. 🙏

**Comment by crobinson-expl** [comment]
> This should now be fixed in `2.6.3`. Let me know if you still encounter any issues. Thanks for the patience. 🙏

Thanks, @adamberecz!
